silverbackis/common-js-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "bundles": {
+        "JsSdkBundle\\JsSdkBundle": ["all"]
```

Comment:
My suggestion is to normalize path.. first of all, it's your realization of JsSdkBundle, probably somebody would like to do bundle with same name, and what to do in this case? conflict will be caused. My suggestion is to add `Silverbackis` before path. Finally you will have `Silverbackis\\JsSdkBundle\\JsSdkBundle`. You can inspire by top bundles, see examples here: https://packagist.org/profile/?q=bundle&p=0
You need to do changes in your bundle first.

Reply:
Cool, understood - I'll do that thanks.